head/src/test/kotlin/io/qalipsis/core/head/jdbc/repository/CampaignRepositoryIntegrationTest.kt

```diff
@@ -2617,4 +2629,158 @@ internal class CampaignRepositoryIntegrationTest : PostgresqlTemplateTest() {
                 }
             }
     }
+
+    @Test
+    fun `should search campaigns in tenant by given status`() =
+        testDispatcherProvider.run {
+            // given
+            val tenant = tenantRepository.save(tenantPrototype.copy(reference = "my-tenant-2"))
+            val user = userRepository.save(UserEntity(username = "John Doe"))
+            val saved = campaignRepository.save(campaignPrototype.copy(end = null, tenantId = tenant.id, name = "The other name 1"))
+            val saved2 = campaignRepository.save(
+                campaignPrototype.copy(
+                    key = "the-key",
+                    name = "The other name 2",
+                    end = null,
+                    tenantId = tenant.id,
+                    configurer = user.id,
+                    result = SCHEDULED
+                )
+            )
+            val saved3 = campaignRepository.save(
+                campaignPrototype.copy(
+                    key = "the-key3",
+                    name = "The other name 3",
+                    end = null,
+                    tenantId = tenant.id,
+                    configurer = user.id,
+                    result = WARNING
+                )
+            )
+            val saved4 = campaignRepository.save(
+                campaignPrototype.copy(
+                    key = "the-key4",
+                    name = "The other name 4",
+                    end = null,
+                    tenantId = tenant.id,
+                    configurer = user.id,
+                    result = ABORTED
+                )
+            )
+            campaignScenarioRepository.saveAll(
+                listOf(
+                    CampaignScenarioEntity(saved.id, name = "scenario 1", minionsCount = 2),
+                    CampaignScenarioEntity(saved2.id, name = "scenario 2", minionsCount = 2),
+                    CampaignScenarioEntity(saved3.id, name = "scenario 3", minionsCount = 2),
+                    CampaignScenarioEntity(saved4.id, name = "scenario 4", minionsCount = 2)
+                )
+            )
+
+            // when + then
+            assertThat(
+                campaignRepository.findAll(
```

Comment:
Please add the argument names on those calls, because the statuses are excluded while the other values are included. This is misleading.